frontend/src/pages/Token/LaunchForm/Confirm/Ekubo.tsx

```diff
@@ -32,40 +33,41 @@ export default function EkuboLaunch({ previous }: LastFormPageProps) {
   // memecoin
   const { data: memecoin, refresh: refreshMemecoin } = useMemecoin()
 
-  // eth price
-  const ethPrice = useEtherPrice()
+  // quote token price
+  const quoteToken = useQuoteToken(quoteTokenAddress)
+  const quoteTokenPrice = useQuoteTokenPrice(quoteTokenAddress)
 
   // team allocation
   const teamAllocationTotalPercentage = useTeamAllocationTotalPercentage(memecoin?.totalSupply)
 
   // team allocation quote amount
   const teamAllocationQuoteAmount = useMemo(() => {
-    if (!ethPrice || !startingMcap || !teamAllocationTotalPercentage || !ekuboFees) return
+    if (!quoteTokenPrice || !startingMcap || !teamAllocationTotalPercentage || !ekuboFees) return
 
-    // mcap / eth_price * (team_allocation / total_supply * (1 + ekuboFees))
+    // mcap / quote_token_price * (team_allocation / total_supply * (1 + ekuboFees))
     return new Fraction(parseFormatedAmount(startingMcap))
-      .divide(ethPrice)
+      .divide(quoteTokenPrice)
       .multiply(teamAllocationTotalPercentage.multiply(parseFormatedPercentage(ekuboFees).add(1)))
-  }, [ethPrice, startingMcap, teamAllocationTotalPercentage, ekuboFees])
+  }, [quoteTokenPrice, startingMcap, teamAllocationTotalPercentage, ekuboFees])
 
   // starting tick
   const i129StartingTick = useMemo(() => {
-    if (!ethPrice || !startingMcap || !memecoin) return
+    if (!quoteToken || !quoteTokenPrice || !startingMcap || !memecoin) return
 
-    // initial price in quote/MEME = mcap / eth price / total supply
+    // initial price in quote/MEME = mcap / quote token price / total supply
     const initalPrice = +new Fraction(parseFormatedAmount(startingMcap))
-      .divide(ethPrice)
-      .multiply(decimalsScale(DECIMALS))
+      .divide(quoteTokenPrice)
+      .multiply(decimalsScale(18))
```

Comment:
Why `18` instead of `DECIMALS`

Reply:
I saw 18 and DECIMALS used interchangeably, and tought 18 is used for meme token decimals, and DECIMALS is used for quote token decimals, seems like it was just the opposite, sorry.